ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -52,7 +52,7 @@ export type TimeSeriesChartProps = PanelProps<TimeSeriesChartOptions>;
 
 export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   const {
-    spec: { queries, thresholds, y_axis },
+    spec: { queries, thresholds, y_axis, __experimentalEChartsOptionsOverride },
```

Comment:
Hey @stephtaylor! I talked to @saminzadeh about this PR and he's planning to look into the `TimeSeriesChartViz` export part of this task when he's back next week. In the meantime, for this PR it'd be great to focus on `__experimentalEChartsOptionsOverride` support for LineChart.tsx and ability to hide TimeSeriesTooltip.

Can you remove the changes in time-series-chart-model.ts and TimeSeriesChartPanel.tsx? And then in LineChart.tsx can you update [these lines](https://github.com/perses/perses/blob/v0.25.0/ui/components/src/LineChart/LineChart.tsx#L201-L203) to:


```typescript
{showTooltip === true && option.tooltip.showContent === false && (
  <TimeSeriesTooltip chartRef={chartRef} chartData={data} wrapLabels={true} pinTooltip={pinTooltip} unit={unit} />
)}
```

Then when you're testing this, make sure __experimentalEChartsOptionsOverride.tooltip.showContent is true. I think this will hopefully be enough short term for setting the correct unit for each series in the tooltip since it will allow you to customize the tooltip [formatter](https://echarts.apache.org/en/option.html#tooltip.formatter) (there is a good example internally of this in timeline's BarChart.tsx)

Reply:
That sounds good to me! 👍🏼

Reply:
😣 So I tested this and some additional changes will be needed, currently the ECharts tooltip gets cut off:

https://user-images.githubusercontent.com/9369625/228897585-7d51a579-a4cd-442f-826c-9821cf772d86.mp4

Our custom TimeSeriesTooltip uses [Portal](https://mui.com/material-ui/react-portal/) so doesn't run into this, I remember adding `overflow: hidden` to Panel.tsx at some point, but need to see what happens when it is removed

Reply:
Adding `tooltip.appendToBody = true` fixes this!
https://echarts.apache.org/en/option.html#tooltip.appendToBody

Reply:
Oh nice find!